vtex/ab-tester
# Use ABTesting API

#### What is the purpose of this pull request?
Make `ab-tester` use new ABTesting API to get, save and delete data about workspaces being tested.

#### What problem is this solving?
We create this new API for two reasons:
 - We don't want anymore to save metadata for A/B test in our database (probably we want to explode it) and now we think it should be better to save it in Metadata (now, after we will move it to Chronos).
 - Use this new API that save and read from Metadata and then already has cache implemented in its responses.

The second one should grant us a performance gain.

#### How should this be manually tested?
Call the routes defined by the app to initialize a test for a workspace and then call the ABTesting API get routes to verify if the data was saved successfully. Repeat this for updating this data and deleting.

[ch11234]

#### Types of changes
